Dockerfile

```diff
@@ -86,10 +86,9 @@ RUN apt-get clean
 RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 # Install crontab to setup jobs
-RUN echo "0 0 * * * /usr/bin/python /code/manage.py generate_tree" >> /code/cronjob
-RUN echo "0 * * * Mon /usr/bin/python /code/manage.py reset_container_limits" >> /code/cronjob
-RUN echo "0 1 * * * /bin/bash /code/scripts/backup_db.sh" >> /code/cronjob
-RUN echo "0 2 * * * /usr/bin/python /code/manage.py cleanup_dummy" >> /code/cronjob
+RUN echo "0 * * * Mon /usr/local/bin/python /code/manage.py reset_container_limits > /var/log/reset_container_limits.log 2>&1 " >> /code/cronjob
```

Comment:
So - methinks we want the default for this to still be daily, having it run once on a Monday would work for smaller registries, but (I think generally) since the limits in the settings are set to a day, we should reset nightly. Thoughts?